principles/equivalence.md

```diff
@@ -0,0 +1,11 @@
+# Equivalence
+
+The following defines when TRAPI components should be considered equivalent.
+
+* **query graph nodes** - when keys and values are identical
```

Comment:
All keys and values?   What if it is a bound node, then should it be similar to KG nodes, i.e. equivalent identifiers?

Reply:
Name too?